android/anc/src/main/java/org/smartregister/fhircore/anc/ui/anccare/details/AncDetailsFragment.kt

```diff
@@ -155,20 +151,28 @@ class AncDetailsFragment : Fragment() {
         binding.txtViewNoCarePlan.visibility = View.VISIBLE
         binding.txtViewCarePlanSeeAllHeading.visibility = View.GONE
         binding.imageViewSeeAllArrow.visibility = View.GONE
-        binding.carePlanListView.visibility = View.GONE
+        binding.txtViewCarePlan.visibility = View.GONE
```

Comment:
You may perhaps consider using the View extension methods `show` or `hide` and use the `apply` scope function to enhance the code readability.

```kotlin
binding.apply {
        txtViewNoCarePlan.show()
        txtViewCarePlanSeeAllHeading.hide()
        imageViewSeeAllArrow.hide()
        txtViewCarePlan.hide()
}
```